### ISSUE TITLE
Accessibility regression introduced in #190

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1591]
It looks like the change from vue-multiselect introduced an accessibility bug with regard to color contrast. Specifically the list highlight color used by the new library is ```#5897fb``` with white text, this fails the contrast requirement of 3:1 (#5897fB : #ffffff comes in at 2.9:1).

**Comment by rmmayo** [comment]
https://webaim.org/resources/contrastchecker

**Comment by rmmayo** [comment]
![image](https://user-image:+1: ithubusercontent.com/47030042/191349303-d6e6d477-30a9-4ea2-bba4-759f77d6fdc9.png)

:+1
